cmd/neofs-ir/defaults.go

```diff
@@ -109,9 +109,7 @@ func defaultConfiguration(cfg *viper.Viper) {
 	cfg.SetDefault("indexer.cache_timeout", 15*time.Second)
 
 	// extra fee values for working mode without notary contract
-	cfg.SetDefault("fee.main_chain", 5000_0000)                  // 0.5 Fixed8
-	cfg.SetDefault("fee.side_chain", 2_0000_0000)                // 2.0 Fixed8
-	cfg.SetDefault("fee.named_container_register", 25_0000_0000) // 25.0 Fixed8
```

Comment:
This one is (incorrectly) mentioned in `pkg/innerring/config/fee.go`, can be adjusted as well.